package.json

```diff
@@ -5,6 +5,7 @@
     "test": "gulp test",
     "clean": "gulp clean",
     "start": "cross-env NODE_ENV=production gulp build+start --obj-dir-name production",
+    "serve": "cross-env NODE_ENV=production gulp build+serve",
```

Comment:
Should specify a dirname, otherwise it'd just use "obj", and actually use a development NODE_ENV. Something like "development-headless".